catalog/spatial/registry/registry-schema-bindings/src/main/java/org/codice/ddf/registry/schemabindings/EbrimConstants.java

```diff
@@ -0,0 +1,190 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.registry.schemabindings;
+
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.ObjectFactory;
+
+/**
+ * This class consists of constants used by the ebrim schema bindings.
+ */
+public final class EbrimConstants {
```

Comment:
The classes that create the maps should be the ones that define those constants. The classes that convert to Objects from the maps should refer to those.